world/physics/models/test_solar_generation.py

```diff
@@ -0,0 +1,189 @@
+from functools import partial
+import yaml
+from tqdm import tqdm
+
+import numpy as np
+from scipy.spatial.transform import Rotation
+from matplotlib import pyplot as plt
+
+import brahe
+from brahe.constants import R_EARTH, GM_EARTH
+from brahe.epoch import Epoch
+
+from world.math.time import increment_epoch
+from world.math.integrators import RK4
+from world.physics.models.gravity import Gravity
+from world.physics.models.solar_generation import SolarGeneration
+
+
```

Comment:
config file uses a date starting May 2024. To use this date, we need brahe to update its Earth rotation database before running. Since time is not of too much concern, consider adding a ``` brahe.utils.download_iers_bulletin_ab(outdir=brahe.constants.DATA_PATH) ```